### PR TITLE
Consolidate app params

### PR DESCRIPTION
**Pull request by beedub**
We exceeded the limit of app params (63 > 60). This PR attempts to reduce the number of cf params generated.

1. Remove the health params altogether
1. Conditionally include Proxy and Secure params only if they're specifically set to true in their corresponding label

So in total, each balancer would have between 2-4 fewer cf params generated for it

### CODE REVIEW

File: api/models/templates/app.tmpl [hunk 1]
```diff
@@ -287,24 +249,28 @@
         "Default" : "{{ index $balancer.Randoms .Balancer }}",
         "Description" : ""
       },
+      {{ if .Secure }}
```
Comment (ddollar) [inline api/models/templates/app.tmpl]: I think these conditionals will be problematic if the parameter is not there at all. We should either fully de-parameterize or make them unconditional I think.

File: api/models/templates/app.tmpl [hunk 2]
```diff
@@ -386,68 +352,59 @@
...
-                          {
-                            "Fn::If": [
-                              "Balancer{{ upper $balancer.ProcessName }}Port{{ .HealthPort }}Secure",
-                              {"Fn::FindInMap": ["PortProtocol", "http", "SecureInstanceProtocol"]},
-                              {"Fn::FindInMap": ["PortProtocol", "http", "InstanceProtocol"]}
-                            ]
-                          }
-                        ]
-                      },
+                      {{ if eq .HealthPath "" }}
```
Comment (ddollar) [inline api/models/templates/app.tmpl]: what is the difference in this branch?

File: api/models/templates/app.tmpl (hunk 1 above)
Comment (beedub) [inline api/models/templates/app.tmpl]: Yeah you're right. I'll fully de-param

Comment (beedub) [inline api/models/templates/app.tmpl]: updated

File: api/models/templates/app.tmpl (hunk 2 above)
Comment (beedub) [inline api/models/templates/app.tmpl]: difference is port protocol.

healthpath set = tcp
healthpath not set = http

**Pull request closed without merging**